.github/workflows/unittest.yml

```diff
@@ -13,7 +13,9 @@ permissions:
 
 jobs:
   unit:
-    runs-on: ubuntu-latest
+    # Use ubuntu-22.04 until Python 3.7 is removed from the test matrix
```

Comment:
Could we add a TODO and link to an umbrella issue for this? A quick search suggests the only currently filed issue for the 3.7 deprecation is #11437, but that deals more with messaging the deprecation. We should have an issue for actually removing support when the time comes, and part of that should be removing this Ubuntu specification.

Reply:
Done in https://github.com/googleapis/google-cloud-python/pull/13377/commits/84387efc591a3162dec24bc64a527a3b556f4b5a